src/coreclr/pal/src/thread/process.cpp

```diff
@@ -3277,19 +3276,37 @@ Return
 BOOL
 PROCAbortInitialize()
 {
-    char* enabled = getenv("COMPlus_DbgEnableMiniDump");
-    if (enabled != nullptr && _stricmp(enabled, "1") == 0)
+    CLRConfigNoCache enabledCfg= CLRConfigNoCache::Get("DbgEnableMiniDump", /*noprefix*/ false, &getenv);
+
+    DWORD enabled = 0;
+    if (enabledCfg.IsSet()
+        && enabledCfg.TryAsInteger(10, enabled)
```

Comment:
These could also be set to 16 with no issue.

Reply:
I don't see the need to change the defaults here. Is there one?

Reply:
No, it was in the case of default radix.